content/en/docs/concepts/cluster-administration/system-metrics.md

```diff
@@ -21,7 +21,7 @@ This format is structured plain text, designed so that people and machines can b
 ## Metrics in Kubernetes
 
 In most cases metrics are available on `/metrics` endpoint of the HTTP server. For components that
-doesn't expose endpoint by default it can be enabled using `--bind-address` flag.
+don't expose endpoint by default, it can be enabled using `--bind-address` flag.
```

Comment:
(nit)
```suggestion
don't expose an endpoint by default, it can be enabled using the `--bind-address` flag.
```

or (better):

```suggestion
don't expose a metrics endpoint by default, you can enable that using the `--bind-address`
command line argument.
```